api/src/wfl/service/datarepo.clj

```diff
@@ -244,32 +251,41 @@
 
 ;; HACK: (str "datarepo_" name) is a hack while accessInformation is nil.
 ;;
-(defn ^:private bq-datasetId-dataProject
-  "Return a BigQuery [datasetId dataProject] pair for `dataset-or-snapshot`."
+(defn ^:private bq-qualification
```

Comment:
I tweaked this some, the ordering in the pair previously returned tripped me up a few times since it was opposite to the BQ path qualification.  We're now returning a map.